server/src/test/java/org/elasticsearch/search/aggregations/metrics/WrapperTDigestArraysTests.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.search.aggregations.metrics;
+
+import org.elasticsearch.common.unit.ByteSizeValue;
+import org.elasticsearch.tdigest.arrays.TDigestArrays;
+import org.elasticsearch.tdigest.arrays.TDigestDoubleArray;
+import org.elasticsearch.tdigest.arrays.TDigestIntArray;
+import org.elasticsearch.test.ESTestCase;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+
+public class WrapperTDigestArraysTests extends ESTestCase {
+    public void testIntEmpty() {
+        try (TDigestIntArray array = intArray(0)) {
+            assertThat(array.size(), equalTo(0));
+        }
+    }
+
+    public void testIntBasicOperations() {
```

Comment:
TO-DO: _WTH are "basic operations"?_ Also, check if there are new array methods without testing

Reply:
Just changed the names, and added extra tests for all classes and methods